### ISSUE TITLE
The main record - projects.yaml

### ISSUE THREAD

**Issue by 0xc0170** [0xc0170/project_generator#49]
This main project should be always in the root directory. I would like to add this requirement.

What about some additional details? I am thinking about adding there option for settings paths, to override default paths. 

```
projects:
  my_project_1:
     1.yaml
settings:
  uv4:
    path-to-u4-on-my-machine
```

**Comment by sg-** [comment]
This makes a mess of the program directory as there will be other python tools so may be best to keep in a tools directory

**Comment by 0xc0170** [comment]
I'll leave it optional. Either to specify the location of projects.yaml via -f argument, or default one will be in the root directory.

**Comment by 0xc0170** [comment]
Closed via the #50
